### PR TITLE
Memorygame xmas update

### PR DESCRIPTION
**Pull request by jejdacz**

### CODE REVIEW

File: pages/kids/memory-game/index.tsx [hunk 1]
```diff
@@ -1,29 +1,50 @@
...
   return (
-    <div className="bg-gradient-to-r from-[#fdf6d2] to-[#99bde4] -mb-8 -m-2">
-      <SEO
-        title={t(`seo.kids_page_memorygame_title.${getCountryVariant()}`)}
-        description={t(`seo.kids_page_memorygame_description.${getCountryVariant()}`)}
-        image="https://www.movapp.cz/icons/movapp-cover-kids.jpg"
-      />
-      <div className="flex flex-wrap flex-col items-center min-h-screen m-auto sm:py-10 py-2 px-2 sm:px-4 overflow-hidden">
-        <MemoryGame />
+    <PhrasesContext.Provider value={kidsCategory.translations}>
```
Comment (JuroUhlar) [inline pages/kids/memory-game/index.tsx:23]: Why do you need `Context` here? Was it not possible to just pass the `kidsCategory.translations` to `<MemoryGame>` ?

Comment (JuroUhlar) [reply pages/kids/memory-game/index.tsx:23]: If its harder than it looks, don't worry, we can simplifiy it later but it's worth a shot :)

Comment (jejdacz) [reply pages/kids/memory-game/index.tsx:23]: I would prefer props too. I know that data goes through two another components ,but it has a simple reason -> Loading only specific css file for each theme. If it was possible to lazy load css module we can move code from Theme components to ThemeLoader.

Comment (m5rk1t) [reply pages/kids/memory-game/index.tsx:23]: I will change it, but I think it's a props drilling)

Comment (JuroUhlar) [reply pages/kids/memory-game/index.tsx]: Yeah we need to refactor the ThemeLoader in the future, for now prop-drilling through 2 components is more readable than Context I think :)

**Review by JuroUhlar — APPROVED** [review]
Good work, thanks! :rocket:

**Pull request merged**